### PR TITLE
osa - add logbridge promotion

### PR DESCRIPTION
**Pull request by mjudeikis**
adds logbridge image promotion for osa

@kargakis ptal if I didn't miss anything

### CODE REVIEW

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml [hunk 1]
```diff
@@ -29,6 +29,17 @@ images:
...
       - e2e-tests
   to: e2e-tests
+- dockerfile_path: Dockerfile.logbridge
+  from: base
+  inputs:
+    bin:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/openshift-azure/logbridge
+    root:
```
Comment (kargakis) [inline ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]: I think we don't need this
```yaml
root:
  as:
  - logbridge
```

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ postsubmits:
         - --artifact-dir=$(ARTIFACTS)
         - --give-pr-author-access-to-namespace=true
         - --secret-dir=/usr/local/e2e-secrets/azure
-        - --target=[output:stable:sync]
+        - --target=[output:stable:sync:logbridge]
```
Comment (kargakis) [inline ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: Interesting. Is this documented somewhere?

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml (hunk 1 above)
Comment (mjudeikis) [reply ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]: i think this is image build destination. we kinda have it for all other images.

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml (hunk 2 above)
Comment (mjudeikis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: This is target list. I know I needed to add sync to be promoted to another stage. We will need in the future enable our e2e to use this new logbridge image too, same as sync.

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml (hunk 1 above)
Comment (kargakis) [reply ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]: @bbguimaraes @droslean can you please verify whether we need `root`? Pretty sure @droslean was saying to me we didn't need it a while back.

File: ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml (hunk 2 above)
Comment (kargakis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: Is this notation documented somewhere in the ci-operator docs?

Comment (mjudeikis) [reply ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-postsubmits.yaml]: Not much :/ `https://github.com/openshift/release/tree/master/ci-operator#using-a-template`
I remember that it was not promoting sync If I didn't have this one...

File: ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml (hunk 1 above)
Comment (droslean) [reply ci-operator/config/openshift/openshift-azure/openshift-openshift-azure-master.yaml]: ci-operator is not expecting that field. For more information check [here](https://github.com/openshift/ci-operator/blob/master/CONFIGURATION.md)
So, you can remove it. The tag for the image is defined in `to` field.

**Pull request merged**